### PR TITLE
shop layers indices

### PR DESCRIPTION
**Pull request by rfrancel**
@ltclm yes I had to correct the indices for the shop layers, strange that it showed no problem for the other ones.. Here is a PR with all the indices for all shop layers.
If you can review this, it would be very nice. Thanks

### CODE REVIEW

**Comment by ltclm** [comment]
Hello
Please replace ``||`` concatenation with ``concat_ws``function:
p.e.
```sql
            , remove_accents(s_map_number||' '||coalesce(s_title_de,'')||' '||coalesce(release::text,'')) as detail 
--with the more compact form
            , remove_accents(concat_ws(' ',s_map_number, s_title_de, release) as detail 
```

File: conf/stopo.conf.part [hunk 1]
```diff
@@ -787,7 +1008,7 @@ source src_ch_swisstopo_hebungsraten : def_searchable_features
       SELECT bgdi_id as id \
```
Comment (ltclm) [inline conf/stopo.conf.part]: This query is not working
```
ERROR:  column "release" does not exist
LINE 5:             , remove_accents(coalesce(release::text,'')) as ...
```

**Comment by rfrancel** [comment]
@AFoletti thank you for checking and merging if ok!

File: conf/stopo.conf.part [hunk 2]
```diff
@@ -678,6 +661,23 @@ source src_ch_swisstopo_wk50-papierkarte_metadata : def_searchable_features
...
 }
 
+source src_ch_swisstopo_wkt33-papierkarte_metadata : def_searchable_features
+{
+   sql_db = stopo_dev
+   sql_query = \
+      SELECT row_number() OVER(ORDER BY pk_product asc) as id \
+          , pk_product as label \
+          , 'feature' as origin \
+          , remove_accents(concat_ws(' ',s_map_number, s_title_de,release::text) as detail \
```
Comment (AFoletti) [inline conf/stopo.conf.part]: missing a ")"

File: conf/stopo.conf.part [hunk 3]
```diff
@@ -780,6 +780,227 @@ source src_ch_swisstopo_sfk300-papierkarte_metadata : def_searchable_features
...
+}
+
+source src_ch_swisstopo_gas25-papierkarte_metadata : def_searchable_features
+{
+   sql_db = stopo_dev
+   sql_query = \
+      SELECT row_number() OVER(ORDER BY pk_product asc) as id \
+          , pk_product as label \
+          , 'feature' as origin \
+          , remove_accents(concat_ws(' ',s_map_number, s_title_de,release::text) as detail \
```
Comment (AFoletti) [inline conf/stopo.conf.part]: missing a ")"

File: conf/stopo.conf.part [hunk 4]
```diff
@@ -780,6 +780,227 @@ source src_ch_swisstopo_sfk300-papierkarte_metadata : def_searchable_features
...
+}
+
+source src_ch_swisstopo_gsk-papierkarte_metadata : def_searchable_features
+{
+   sql_db = stopo_dev
+   sql_query = \
+      SELECT row_number() OVER(ORDER BY pk_product asc) as id \
+          , pk_product as label \
+          , 'feature' as origin \
+          , remove_accents(concat_ws(' ',s_map_number, s_title_de,release::text) as detail \
```
Comment (AFoletti) [inline conf/stopo.conf.part]: ")" missing

File: conf/stopo.conf.part [hunk 5]
```diff
@@ -780,6 +780,227 @@ source src_ch_swisstopo_sfk300-papierkarte_metadata : def_searchable_features
...
+}
+
+source src_ch_swisstopo_ggk200-papierkarte_metadata : def_searchable_features
+{
+   sql_db = stopo_dev
+   sql_query = \
+      SELECT row_number() OVER(ORDER BY pk_product asc) as id \
+          , pk_product as label \
+          , 'feature' as origin \
+          , remove_accents(concat_ws(' ',s_map_number, s_title_de,release::text) as detail \
```
Comment (AFoletti) [inline conf/stopo.conf.part]: ")" missing

**Comment by AFoletti** [comment]
thanks! LGTM now!

**Pull request merged**